litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/ChooseWorkflowFromExisting.tsx

```diff
@@ -123,17 +165,42 @@ const ChooseWorkflowFromExisting = () => {
                             className={classes.templateIconBg}
                           />
                         </div>
+                        <IconButton
+                          onClick={() => {
+                            deleteTemplate({
+                              variables: { data: templateData.template_id },
+                            });
+                          }}
+                        >
+                          <img
+                            alt="delete"
+                            src="./icons/deleteBox.svg"
+                            height="30"
```

Comment:
please add a unit

Reply:
30 px -> 1.8 rem

Reply:
Unit is not required in `img` tag. It takes pixel by default.
For reference https://www.w3schools.com/tags/att_img_height.asp#:~:text=The%20height%20attribute%20specifies%20the,when%20the%20page%20is%20loaded

Reply:
It would have been `style={{height: "1.8rem'}}` but in this case default pizel values would do.